kedro-datasets/CONTRIBUTING.md

```diff
@@ -66,14 +102,19 @@ We use a branching model that helps us keep track of branches in a logical, cons
 | `fix`            | Non-breaking change which fixes an issue                                    |
 | `tests`          | Changes to project unit (`tests/`) and / or integration (`features/`) tests |
 
-## Plugin contribution process
+## Dataset contribution process
 
  1. Fork the project
  2. Develop your contribution in a new branch.
- 3. Make sure all your commits are signed off by using `-s` flag with `git commit`.
- 4. Open a PR against the `main` branch and sure that the PR title follows the [Conventional Commits specs](https://www.conventionalcommits.org/en/v1.0.0/) with the scope `(datasets)`.
- 5. Make sure the CI builds are green (have a look at the section [Running checks locally](#running-checks-locally) below)
- 6. Update the PR according to the reviewer's comments
+ 3. Add your dataset to the appropriate location:
+    - `kedro-datasets/experimental` for experimental datasets.
+    - `kedro-datasets/kedro_datasets` for datasets meeting regular standards.
+ 4. Make sure all your commits are signed off by using `-s` flag with `git commit`.
+ 5. Open a PR against the `main` branch and sure that the PR title follows the [Conventional Commits specs](https://www.conventionalcommits.org/en/v1.0.0/) with the scope `(datasets)`.
```

Comment:
I think the word "make" is missing here? 

"Open a PR against the `main` branch and **make** sure that..."